src/connectionmanager.js

```diff
@@ -936,29 +761,8 @@ export default class ConnectionManager {
 
             const credentials = credentialProvider.credentials();
             options = options || {};
```

Comment:
I would remove `afterConnectValidated` and migrate all the logic to `onSuccessfulConnection` since the latter is just a stub method now.